Sources/ScipioKit/ProjectGenerator/ProjectGenerator.swift

```diff
@@ -0,0 +1,505 @@
+                }
+
+            let linkReferences: [PBXBuildFile]
+            if target.type == .library {
+                linkReferences = dependsTargets
+                    .compactMap { xcodeTargets[$0] }
+                    .map { dependency in
+                        addObject(PBXBuildFile(file: dependency.product))
+                    }
+            } else {
+                linkReferences = []
+            }
+
+            let linkPhase = addObject(
+                PBXFrameworksBuildPhase(files: linkReferences)
+            )
+            xcodeTarget.buildPhases.append(linkPhase)
+
+            // Add -fmodule-map-file for Swift targets
+            let isSwiftTarget = target.underlyingTarget is SwiftTarget
+            if isSwiftTarget {
+                for dependency in dependsTargets {
+                    guard let moduleMapPath = moduleMaps[dependency] else { continue }
+                    let relativePath = moduleMapPath.relative(to: sourceRoot!)
+                    xcodeTarget.buildConfigurationList?.buildConfigurations.forEach { configuration in
+                        configuration.buildSettings = configuration.buildSettings.appending(
+                            "OTHER_SWIFT_FLAGS",
+                            values: "-Xcc", "-fmodule-map-file=$(SRCROOT)/\(relativePath.pathString)"
+                        )
+                    }
+                }
+            }
+        }
+    }
+
+    private func makeTarget(for target: ResolvedTarget) throws -> PBXNativeTarget {
+        let targetSettingsGenerator = TargetBuildSettingsGenerator(
+            package: package,
+            isDebugSymbolsEmbedded: buildOptions.isDebugSymbolsEmbedded,
+            isStaticFramework: buildOptions.frameworkType == .static
+        )
+
+        guard let productType = target.xcodeProductType else {
+            throw Error.unsupportedTarget(targetName: target.name, kind: target.type)
+        }
+
+        // Generate Info.plist
+        let plistPath = package.buildDirectory.appendingPathComponent(target.infoPlistFileName)
+        fileSystem.write(infoPlist.data(using: .utf8)!, to: plistPath)
+
+        let buildConfigurationList = addObject(
+            XCConfigurationList(buildConfigurations: [
+                addObject(targetSettingsGenerator.generate(for: target, configuration: .debug, infoPlistPath: plistPath)),
+                addObject(targetSettingsGenerator.generate(for: target, configuration: .release, infoPlistPath: plistPath)),
+            ])
+        )
+
+        let productRef: PBXFileReference?
+        if let productGroup = pbxProj.rootObject?.productsGroup {
+            productRef = try productGroup.addFile(
+                at: target.productPath.toPath(),
+                sourceTree: .buildProductsDir,
+                sourceRoot: target.sources.root.toPath(),
+                validatePresence: false
+            )
+        } else {
+            productRef = nil
+        }
```

Comment:
The if statement can be removed.
```suggestion
        let productRef: PBXFileReference? = try? pbxProj.rootObject?
            .productsGroup?
            .addFile(
                at: target.productPath.toPath(),
                sourceTree: .buildProductsDir,
                sourceRoot: target.sources.root.toPath(),
                validatePresence: false
            )
```